.github/actions/spell-check/whitelist/alphabet.txt

```diff
@@ -1,3 +1,4 @@
+AAAAAAAAAA
```

Comment:
Man, maybe we need the pattern of 
```
([A-Za-z])\1{3,}
```
To match any letter repeated 4 or more times.

Reply:
If that works, go for it!
I don't use regexps that fancy.

Reply:
Oh god, regexes with backreferences. I get that we want to do it because they have to be the same character, but man, that's scary.